app/routes/events/view/tickets/add-order.js

```diff
@@ -1,15 +1,25 @@
 import Route from '@ember/routing/route';
+import EmberTableRouteMixin from 'open-event-frontend/mixins/ember-table-route';
 
-export default Route.extend({
+export default class extends Route.extend(EmberTableRouteMixin) {
   titleToken() {
     return this.l10n.t('Add Order');
-  },
-  async model() {
+  }
+
+  async model(params) {
     const eventDetails = this.modelFor('events.view');
-    let queryObject = {};
+    const searchField = 'name';
+    let filterOptions = [];
+    filterOptions = this.applySearchFilters(filterOptions, params, searchField);
+    let queryString = {
+      filter         : filterOptions,
+      'page[size]'   : params.per_page || 10,
+      'page[number]' : params.per_page || 1
+    };
+    queryString = this.applySortFilters(queryString, params);
     return {
-      tickets    : await eventDetails.query('tickets', queryObject),
-      query      : queryObject,
+      tickets    : await this.asArray(eventDetails.query('tickets', queryString)),
```

Comment:
@CosmicCoder96 Since I changed the tickets property to  `asArray`, All the function using tickets are broke as they were using `forEach` and `toArray` helpers with `model.tickets`, How do I tackle this?

Reply:
@CosmicCoder96 Since I changed the tickets property to asArray, All the function using tickets are broke as they were using forEach and toArray helpers with model.tickets, How do I tackle this?